### DISCUSSION TITLE
Errors occured during template validation

### DISCUSSION THREAD

**Discussion by Jaypanchal1401** [projectdiscovery/nuclei#2617] [General]
i did able to validate template it shows the this error 
Could not validate templates: errors occured during template validation

**Comment by ehsandeep** [comment]
@Jaypanchal1401 that means, you are using templates that are not compatible or needs to be updated.

You can learn about error and what to change by using `-validate` flag which should show the exact part of error or line.

**Reply by Jaypanchal1401** [reply]
but I validate online its valid templates if you have a query then I will send a link to the templates below.
https://github.com/pikpikcu/nuclei-templates

**Reply by ehsandeep** [reply]
@Jaypanchal1401 with `-validate` it checks if the template is compatible with the latest nuclei engine and not related to yaml syntax.

**Reply by Jaypanchal1401** [reply]
i checked with this command  and it gives this below error
Could not validate templates: errors occured during template validation
what should I do?

**Reply by ehsandeep** [reply]
@Jaypanchal1401 use `-v` flag to see the errors and update the template.

**Reply by Jaypanchal1401** [reply]
i updated the template but after same error is coming
